.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+
+### Changes
+
+Describe the key changes in this PR with the Jira Issue reference 
+
+### Screenshots (if any)
+
+Attach screenshots to illustrate changes, if applicable
```

Comment:
very very few cases where we need screenshots for the changes
Perhaps we can update this to changes to public api or something else